tests/test_schema_validator.py

```diff
@@ -35,14 +35,17 @@ def mock_schema(tmpdir):
     with open(schema_file_path, "w") as schema_file:
         json.dump(schema_content, schema_file)
 
-    SCHEMA_MAPPING[TestJobType.TEST_BENCHMARK] = schema_file_path
-    return SCHEMA_MAPPING
+    SCHEMA_MAPPING = {
+        TestJobType.TEST_BENCHMARK: str(schema_file_path),
+    }
+    with patch("metriq_gym.benchmarks.SCHEMA_MAPPING", SCHEMA_MAPPING):
+        yield
 
 
 @pytest.fixture
 def valid_params():
     return {
-        "benchmark_name": TEST_BENCHMARK_NAME,
+        "title": TEST_BENCHMARK_NAME,
```

Comment:
One fix would be to change `title` -> `benchmark_name`, but this won't completely fix things as the `TEST_BENCHMARK_NAME` is not one of the recognized names that we check for, and therefore will fail. Do we want to relax this error checking?